README.md

```diff
@@ -92,15 +92,21 @@ with an existing Windows installation.
 ##### Development VM Setup
 
 1. Download a Microsoft Windows 10 [development virtual machine].
-2. Open a privileged PowerShell prompt (hit Windows Key + `X` and open
-   `Windows PowerShell (Admin)`).
+2. Open a PowerShell prompt (hit Windows Key + `X` and open
+   `Windows PowerShell`).
 3. Run the [automated setup script]:
    ```powershell
    Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
 
    iwr -useb 'https://github.com/rancher-sandbox/rancher-desktop/raw/main/scripts/windows-setup.ps1' | iex
```

Comment:
The setup script tries to install Visual Studio components, which _does_ require admin.